backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -958,4 +958,79 @@ databaseChangeLog:
               - column:
                   name: date_tested_backdate
                   type: DATETIME
-                  remarks: timestamp to store a user correction to when the test was ordered
\ No newline at end of file
+                  remarks: timestamp to store a user correction to when the test was ordered
+  - changeSet:
+      id: add-data-hub-upload
+      author: tnielsen2@cdc.gov
+      comment: Add data_hub_upload table used to the automated results uploader.
+      changes:
+        - sql:
+            remarks: Create the enumerations needed for DataHubUpload.
+            sql: |
+              CREATE TYPE ${database.defaultSchemaName}.DATA_HUB_UPLOAD_STATUS as ENUM('IN_PROGRESS', 'SUCCESS', 'FAIL');
+            rollback:
+              sql: |
+                DROP TYPE ${database.defaultSchemaName}.DATA_HUB_UPLOAD_STATUS;
+        - createTable:
+            tableName: data_hub_upload
+            remarks: Track uploads and results of test_events to the data-hub
+            columns:
+              - column:
+                  name: internal_id
+                  type: *idtype
+                  remarks: The internal database identifier for this entity.
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: job_status
+                  type: ${database.defaultSchemaName}.DATA_HUB_UPLOAD_STATUS
+                  remarks: State of this upload. Enumerated value
+                  constraints:
+                    nullable: false
+              - column:
+                  name: created_at
+                  type: DATETIME
+                  remarks: The creation timestamp for this entity.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: updated_at
+                  type: DATETIME
+                  remarks: The timestamp for the most recent update of this entity.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: records_processed
+                  type: int
+                  remarks: Number of rows sent to the server
+              - column:
+                  name: error_message
+                  type: text
```

Comment:
Can be `*string`, but possibly it's worth leaving this way to call out that it is actually expected to hold a longer value.